Intersect (Core)/Config/PlayerOptions.cs

```diff
@@ -16,10 +16,15 @@ public partial class PlayerOptions
         public bool AllowCombatMovement { get; set; } = true;
 
         /// <summary>
-        /// Enables de client setting 'Auto-turn to target'.
+        /// Enables the client feature 'Auto-turn to target'.
         /// </summary>
         public bool EnableAutoTurnToTarget { get; set; } = true;
 
+        /// <summary>
+        /// Sets the delay (milliseconds) for the feature 'Auto-turn to target'.
+        /// </summary>
+        public long AutoTurnToTargetDelay { get; set; } = 50;
```

Comment:
Honestly this should be at like 500ms not 50. Also can be a `ushort` not a `long`.